packages/experimental/Divider/src/Divider.types.ts

```diff
@@ -14,56 +14,67 @@ export interface DividerProps {
   /**
    * If a text or icon is passed, this dictates where content appears in the divider: at the start, centered, or towards the end.
    * @default 'center'
+   * @platform win32
```

Comment:
Rather than this, could we add a note saying it doesn't apply to mobile? Reason being I imagine it might still apply to windows/macOS.

Reply:
It makes sense , added that.